.github/ISSUE_TEMPLATE/~release-checklist.md

```diff
@@ -30,8 +30,8 @@ about: Checklist for core developers to complete as part of making a release
 * [ ] Verify there is a new [Zenodo DOI](https://doi.org/10.5281/zenodo.1169739) minted for the release.
    - [ ] Verify that the new release archive metadata on Zenodo matches is being picked up as expected from [`CITATION.cff`](https://github.com/scikit-hep/pyhf/blob/master/CITATION.cff).
 * [ ] Verify that a Binder has properly built for the new release.
-* [ ] Watch for a GitHub notification that there is an automatic PR to the
-  [Conda-forge feedstock](https://github.com/conda-forge/pyhf-feedstock). This may take multiple hours to happen. If there are any changes needed to the Conda-forge release make them **from a personal account** and not from an organization account to have workflows properly trigger.
+* [ ] Watch for a GitHub notification that there is an automatic PR to the [Conda-forge feedstock](https://github.com/conda-forge/pyhf-feedstock). This may take multiple hours to happen. If there are any changes needed to the Conda-forge release make them **from a personal account** and not from an organization account to have workflows properly trigger.
+   - [ ] Check if any requirements need to be updated by commenting "@conda-grayskull show requirements" on the PR.
```

Comment:
This bit is kinda awesome. https://twitter.com/mdtrevisani/status/1487894867799859201

Reply:
@marcelotrevisani given https://github.com/conda-forge/staged-recipes/pull/19135#issuecomment-1239350215 would you recommend holding off on adding use of the bot as part of a release process?

Reply:
Hi, I'm planning to make some modifications next week and it should improve things. If you want to wait a bit you can ping me in a few weeks to see how it's going. Otherwise, I recommend to use the bot

Reply:
That's fine! I was just wanting to make sure that the bot wasn't going to be deprecated before putting it into our release procedure checklist.

Reply:
Migrating all of this to PR #2021.